### PR TITLE
D8UN-1793 & D8UN-1796

### PR DESCRIPTION
**Pull request by AdamMcFet**

### CODE REVIEW

File: project/sites/communityrelations/themes/communityrelations_theme/src/scss/components.scss [hunk 1]
```diff
@@ -35,6 +52,14 @@
     padding: 1rem 2.8rem;
     border-radius: 5px;
 
+    @media screen and (max-width: $tab) {
+      font-size: 1.6rem;
```
Comment (soda736) [inline project/sites/communityrelations/themes/communityrelations_theme/src/scss/components.scss:56]: You should use this @include adjust-font-size-to for all your font sizes.

File: project/sites/communityrelations/themes/communityrelations_theme/src/scss/components.scss [hunk 2]
```diff
@@ -35,6 +52,14 @@
...
+      display: inline;
+    }
+    @media screen and (max-width: $mob) {
+      font-size: 1.2rem
```
Comment (soda736) [inline project/sites/communityrelations/themes/communityrelations_theme/src/scss/components.scss:60]: You should use this @include adjust-font-size-to for all your font sizes.

**Review by soda736 — CHANGES_REQUESTED** [review]

File: project/sites/communityrelations/themes/communityrelations_theme/src/scss/components.scss (hunk 1 above)
Comment (AdamMcFet) [reply project/sites/communityrelations/themes/communityrelations_theme/src/scss/components.scss]: Done. thank you Adam

**Review by soda736 — APPROVED** [review]
LG

**Pull request merged**